src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEHeaderBuilder.cs

```diff
@@ -105,7 +105,7 @@ public static PEHeaderBuilder CreateLibraryHeader()
             return new PEHeaderBuilder(imageCharacteristics: Characteristics.ExecutableImage | Characteristics.Dll);
         }
 
-        internal bool Is32Bit => Machine != Machine.Amd64 && Machine != Machine.IA64 && Machine != Machine.Arm64;
+        internal bool Is32Bit => Machine != Machine.Amd64 && Machine != Machine.IA64 && Machine != Machine.Arm64 && Machine != Machine.RiscV64;
```

Comment:
This is not needed. https://github.com/dotnet/runtime/pull/90203#discussion_r1310564046

Reply:
Thanks!